dev_guide/new_app.adoc

```diff
@@ -545,13 +545,20 @@ To see all of the configuration options, click the "Show advanced build and depl
 
 ifdef::openshift-online[]
 == Troubleshooting
-`*EmptyDir*` volumes are disabled in OpenShift Online at present. In some situations new-app will automatically create or use templates which contain `*EmptyDir*` volumes. In this scenario you will need to use `oc edit` to modify the relevant PodSpec and replace the use of `*EmptyDir*` volumes with a `*PersistentVolumeClaim*`.
+Currently, `*EmptyDir*` volumes are disabled in OpenShift Online. In some
+situations, `oc new-app` will automatically create or use templates which
```

Comment:
Do not write "will" unless you plan to indicate "when" something will happen. Suggested edit:

s/will automatically create or use/automatically creates or uses